js-pkg/create-y-sweet-app/src/run.js

```diff
@@ -0,0 +1,63 @@
+import { exec } from 'node:child_process'
+import { cp, readdir } from 'node:fs/promises'
+import { dirname, resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { parseArgs } from 'node:util'
+
+const { values, positionals } = parseArgs({ options: { template: { type: 'string', short: 't' } } })
```

Comment:
It looks like `parseArgs` throws when it encounters unexpected command line flags. Maybe we should wrap in a try/catch so we can show the `help()` output instead of this error output?:

```
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_PARSE_ARGS_UNKNOWN_OPTION]: Unknown option '--help'
    at new NodeError (node:internal/errors:405:5)
    at checkOptionUsage (node:internal/util/parse_args/parse_args:98:11)
    at node:internal/util/parse_args/parse_args:360:9
    at Array.forEach (<anonymous>)
    at parseArgs (node:internal/util/parse_args/parse_args:357:3)
    at file:///Users/taylor-drifting/code/jamsocket/y-sweet/js-pkg/create-y-sweet-app/src/run.js:7:33
    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:337:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:106:12) {
  code: 'ERR_PARSE_ARGS_UNKNOWN_OPTION'
}

Node.js v18.20.2
```

Reply:
Looks like there's a `strict` option to not throw which still attempts to parse them. `node src/run.js --asdf foo --template nextjs --blah=qwerty` results in this:

```
{
  values: {
    asdf: true,
    template: 'nextjs',
    blah: 'qwerty'
  },
  positionals: [ 'foo' ]
}
```

Think that's okay (silently ignore them) or would you rather just display the help text?

Reply:
Ahh - yeah, maybe it's fine to just turn off strict mode.

Reply:
Took your suggestion and bailed out if they enter unknown flags.